test/mesh.js

```diff
@@ -140,4 +140,56 @@ describe('Mesh', function() {
     });
   });
 
+  describe('Test tangents and bitangent calculation', function () {
+    const data = `
+    v 0 0 0
+    v 0 1 0
+    v 1 0 0
+
+    vn 0 0 1
+
+    vt 0.0 0.0
+    vt 0.1 0.1
+
+    f 1/1/1 2/2/1 3/2/1
+    `
+
+    const m = new Mesh(data);
+    m.calculateTangentsAndBitangents();
+    const tangents = m.tangents;
+    const bitangents = m.bitangents;
+    const normals = m.vertexNormals;
+
```

Comment:
What are your thoughts on adding an extra test to assert that `.tangents` and `.bitangents` both have the same length? In the event that the implementation is ever changed, having a test like that might help provide some insight to any problems.

Reply:
Yeah that should always be the case, but I agree that it makes sense to add the test for reference and if the implementation changes.